docs/apis-tools/console-api/tutorial.md

```diff
@@ -0,0 +1,263 @@
+---
+id: console-api-tutorial
+title: Tutorial
+slug: /apis-tools/console-api/tutorial
```

Comment:
Oh, I just realized! These should probably say `administration-api` instead of `console-api`....and the file path should change...and there are a couple places in the content where we say "Console API" instead of "Administration API".